source/tutorial/react-native.txt

```diff
@@ -192,15 +200,16 @@ computer:
                   
                   npm run android
 
-      .. tip::
-         
-         If you're encountering an error or otherwise having issues,
-         it's likely an issue with how your local environment is set up.
-         Check out the official React Native
-         `development environment setup
-         <https://reactnative.dev/docs/environment-setup>`_
-         guide and ensure that you've followed all the steps for
-         your Development OS and Target OS.
+               If you encounter an error or otherwise haveissues, make sure that
```

Comment:
Looks like there's a typo:

```suggestion
               If you encounter an error or otherwise have issues, make sure that
```

But more broadly, this tip [on the current version of the page](https://www.mongodb.com/docs/atlas/app-services/tutorial/react-native/#build-the-app) is _outside_ of the Android tab, and I assume the advice about making sure your React Native environment is set up correctly is applicable to both iOS _and_ Android. So maybe the tip should remain, and only the paragraph about the `.gradlew` error should be included within the Android tab content?

Reply:
Great point! I moved the info about the RN environment to above the tabs.